src/index.js

```diff
@@ -75,6 +77,19 @@ export default class SaucelabsConnector {
         if (timeout)
             initParams.maxDuration = timeout;
 
+
+        // NOTE: if starts IE 11, the option "Display intranet sites in Compatibility View" should be disabled.
+        // We do this via 'prerun' parameter, which should run our script on Sauce Labs side, before the browser starts.
+        if (browser.browserName === 'internet explorer' && browser.version === '11.0') {

```

Comment:
We should check `browser.browserName` in lower-case.
And I suppose it's possible to use `browser.version = 11` to configure saucelabs? If so, we should check that browser.version contains `11` here or use regexp like `/^11(.0)?/`